src/Resources/ResourceManager/Implementation/Policy/PolicyHelpStrings.cs

```diff
@@ -124,5 +124,39 @@ public static class PolicyHelpStrings
         public const string SetPolicySetDefinitionManagementGroupHelp = "The name of the management group of the policy set definition to update.";
         public const string SetPolicySetDefinitionSubscriptionIdHelp = "The subscription ID of the policy set definition to update.";
         public const string SetPolicySetDefinitionInputObjectHelp = "The policy set definition object to update that was output from another cmdlet.";
+
+        /// <summary>
+        /// Policy exemption cmdlet parawmeter help strings
+        /// </summary>
+        public const string GetPolicyExemptionNameHelp = "The name of the policy exemption to get.";
+        public const string GetPolicyExemptionScopeHelp = "The scope of the policy exemption to get, e.g. /providers/managementGroups/{managementGroupName}, defaults to current subscription.";
+        public const string GetPolicyExemptionIdHelp = "The fully qualified policy exemption ID to get, including the scope, e.g. /subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Authorization/policyExemptions/{policyExemptionName}.";
+        public const string GetPolicyExemptionFilterHelp = "Limits the list of returned policy exemptions to those assigning the policy assignment identified by this fully qualified Id.";
+        public const string GetPolicyExemptionIncludeDescendentsHelp = "Causes the list of returned policy exemptions to include all exemptions related to the given scope, including those from ancestor scopes and those from descendent scopes.";
+        public const string GetPolicyExemptionDoesNothingHelp = "This parameter is ignored if provided with -Name or -Id parameters.";
+        public const string NewPolicyExemptionNameHelp = "The name of the new policy exemption.";
+        public const string NewPolicyExemptionScopeHelp = "The scope of the new policy exemption, e.g. /providers/managementGroups/{managementGroupName}, defaults to current subscription.";
+        public const string NewPolicyExemptionCategoryHelp = "The policy exemption category of the new policy exemption. Possible values are Waiver and Mitigated.";
+        public const string NewPolicyExemptionDisplayNameHelp = "The display name for the new policy exemption.";
+        public const string NewPolicyExemptionDescriptionHelp = "The description for the new policy exemption.";
+        public const string NewPolicyExemptionExpiresOnHelp = "The expiration date and time (in UTC ISO 8601 format yyyy-MM-ddTHH:mm:ssZ) of the new policy exemption.";
+        public const string NewPolicyExemptionPolicyAssignmentIdHelp = "The referenced policy assignment Id for the new policy exemption.";
+        public const string NewPolicyExemptionPolicyDefinitionReferenceIdsHelp = "The policy definition reference ID list when the associated policy assignment is an assignment of a policy set definition.";
```

Comment:
More of a question, but we say "policy set" instead of initiative a lot.  If that's the standard we want to stick with in the SDK, that's fine.  Maybe "policy set (initiative)" going forward to help make it clearer?

Reply:
policy set (initiative) sounds good